src/os/filestore/FileJournal.cc

```diff
@@ -1217,9 +1217,11 @@ void FileJournal::write_thread_entry()
       // but should be fine given that we will have plenty of aios in
       // flight if we hit this limit to ensure we keep the device
       // saturated.
+      uint64_t min_size = 64 * 1024;
       while (aio_num > 0) {
 	int exp = MIN(aio_num * 2, 24);
 	long unsigned min_new = 1ull << exp;
+	min_new = min_new < min_size ? min_size : min_new;
```

Comment:
(actually, for the above, can just change the while condition to while (aio_num >= 2)